mailing_list/models.py

```diff
@@ -106,7 +134,7 @@ class MailingList(models.Model):
     ACCESS_LEVEL_STAFF = 'staff'
 
     canvas_course_id = models.IntegerField()
-    section_id = models.IntegerField()
+    section_id = models.CharField(max_length=32)
```

Comment:
Ugh, I know we talked about this on Friday, but now that I'm actually looking at the code, I'm thinking you original approach is better. Leave this as an IntegerField and set the all course fake section id to 0. Sorry.